calculator.js

```diff
@@ -0,0 +1,20 @@
+var calculator = {
+  addition: function (num1, num2) {
+    return num1 + num2;
+  },
+  subtraction: function (num1, num2) {
+    return (num1 - num2);
+  },
+  multiply: function (num1, num2) {
+    return num1 * num2;
+  },
+  division: function (num1, num2) {
+    return num1 / num2;
+  },
+  exponents: function (num1, num2) {
+    return Math.pow(num1, num2);
+  },
+  squareRoot: function (num1) {
+    return Math.sqrt(num1);
+  }
+};
```

Comment:
Excellent job on this assignment!! :star2: Consider tackling the bonus next time!